dask/dataframe/tests/test_dataframe.py

```diff
@@ -432,6 +432,18 @@ def test_clip(lower, upper):
     assert_eq(ds.clip_upper(upper), s.clip_upper(upper))
 
 
+def test_squeeze():
+    df = pd.DataFrame({'x': [1, 3, 6],
```

Comment:
@mrocklin happy to parametrize this test for dataframes and series if you think that looks cleaner? Not sure what conventions `dask` is following.

Reply:
No strong thoughts on parametrization here.  This function should be simple enough that serious parametrization may not be ncessary.  It looks like we should try this on `df[['x']]` though?